apps/drec-api/src/pods/organization/organization.entity.ts

```diff
@@ -73,43 +67,42 @@ export class Organization
   vatNumber: string;
 
   @ApiProperty({ type: String })
-  @Column()
+  @Column({ nullable: true })
   @IsString()
   signatoryFullName: string;
```

Comment:
So, if this is nullable, then it should be `signatoryFullName: string | null;` and for other fields as well, right?

Reply:
I can't add this to the entity property as I get a postgress issue. So if I put `signatoryFullName: string | null` I get:
`dataTypeNotSupportedError: Data type "Object" in "Organization.signatoryFullName" is not supported by "postgres" database.`

Reply:
Apparently this a common problem with typeorm when adding `nullable: true`.  One solution which I don't like would be:
```
@Column({
    type: 'text',
    unique: true,
    nullable: true,
})
signatoryFullName!: string;
```

I say we keep it like this at works fine on both backend and frontend with nullable values without any linting issues or errors.